reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md

```diff
@@ -35,12 +43,6 @@ All packages are available on our GitHub [releases][] page. After the package is
 [snap]: #snap-package
 [tar]: #binary-archives
 
-> [!TIP]
-> If you already have the [.NET Core SDK](/dotnet/core/sdk) installed, it’s easy to install PowerShell as a [.NET Global tool](/dotnet/core/tools/global-tools).
->
-> ```
-> dotnet tool install --global PowerShell
-> ```
```

Comment:
Are there plans to no longer support installing PowerShell as a global tool?

Reply:
That information was just moved to the bottom of the article.

Reply:
Thanks. I was used to seeing it at the top and missed it in the sidebar.